sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -758,6 +758,11 @@ object SQLConf {
     .intConf
     .createWithDefault(200)
 
+  val THRIFTSERVER_RESULT_ESCAPE_STRUCT_STRING = buildConf("spark.sql.thriftserver.result.escapeStructString.enabled")
```

Comment:
`GitHub Action` seems to complain for this line. Could you check by running `dev/scalastyle`?

Reply:
Fixed. Thanks